pages/reference/OS/updates/update-process.md

```diff
@@ -58,6 +58,11 @@ If the fingerprint check succeeds, the updater will use **Docker** to pull the t
 
 From here, the original updater script takes back control, and finishes up any other changes required by the update. When all this is done, the device is rebooted into the new system.
 
+## Automatic rollbacks
+Rollbacks is a framework designed to automatically roll back the OS update in case something goes wrong.
+
+There are two rollback mechanisms in the OS, covering different update failure modes: one based on health checks, the rollback-health, and another recognizing if the new system is unbootable for some reason, the rollback-altboot. Their detailed operations are explained [here].
```

Comment:
```suggestion
There are two rollback mechanisms in the OS, covering different update failure modes. One based on health checks called the `rollback-health`, and the recognizing if the new system is unbootable for some reason, the `rollback-altboot`. Learn more about these [rollback mechanisms][rollback-mechanism].
```

Reply:
Learn more about adding links to docs the right way here https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#adding-links-to-the-docs